pkgs/servers/monitoring/prometheus/default.nix

```diff
@@ -31,10 +31,10 @@
 }:
 
 let
-  version = "2.40.3";
+  version = "2.41.0";
   webUiStatic = fetchurl {
     url = "https://github.com/prometheus/prometheus/releases/download/v${version}/prometheus-web-ui-${version}.tar.gz";
-    sha256 = "sha256-dvMts9uJNLSp8Qho+yKMLPTy/1c2RgfeEn3UQLIZNc4=";
+    sha256 = "sha256-0EFeAuhQUu+4TCoHsWHTbWTGJLjS37POacP1K42sGqI=";
   };
 in
 buildGoModule rec {
```

Comment:
Right now, you only updated the package version and UI. Please, update the sha256 and vendorSha256 of buildGoModule as well. You can find an example https://github.com/NixOS/nixpkgs/commit/cdcfc6b2f0197f5802bab5b79db95fc96efccc44

Thank you!

Reply:
oh, i missed that bit. Should be fixed now.